server/src/main/java/io/littlehorse/common/util/WfSpecUtil.java

```diff
@@ -0,0 +1,55 @@
+package io.littlehorse.common.util;
+
+import com.google.protobuf.Timestamp;
+import io.littlehorse.common.model.getable.global.wfspec.WfSpecModel;
+import io.littlehorse.common.model.getable.global.wfspec.thread.ThreadVarDefModel;
+import io.littlehorse.sdk.common.proto.WfSpec;
+import io.littlehorse.sdk.common.proto.WfSpecId;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.Map;
+import java.util.Set;
+
+public class WfSpecUtil {
```

Comment:
suggestion:

add a private constructor for this class to prevent class instances since it only contains static methods